data/algorithm_config.yml

```
@@ -0,0 +1,34 @@
+# Input params for OGC application package generator
+# The information provided in this file supports CWL and OGC best practices
+algorithm_description: This application is designed to process Synthetic Aperture Radar (SAR) data from Sentinel-1 GRD (Ground Range Detected) products using a Digital Elevation Model (DEM) obtained from Copernicus.
+algorithm_name: sardem-sarsen
+algorithm_version: mlucas/nasa_ogc
+keywords: ogc, sar
+code_repository: https://github.com/MAAP-Project/sardem-sarsen.git
+citation: https://github.com/MAAP-Project/sardem-sarsen.git
+author: arthurduf
+contributor: arthurduf
+license: https://github.com/MAAP-Project/sardem-sarsen/blob/main/LICENSE
+release_notes: None
+run_command: /app/sardem-sarsen/sardem-sarsen.sh
+ram_min: 5 # mebibytes
+cores_min: 1
+outdir_max: 20 # mebibytes
+
+inputs:
+  - name: bbox
+    doc: Bounding box as 'LEFT BOTTOM RIGHT TOP'
+    label: bounding box
+    type: string
+  - name: stac_catalog_folder
+    doc: STAC catalog folder
+    label: catalog folder
+    type: Directory
+  - name: stac_asset_name
+    doc: STAC asset name
+    label: asset name
+    type: string?
+
+outputs:
+  - name: out
+    type: Directory
```

We need an additional field called glob here to let user specify the output pattern. Something that could be address in the next revision if you want to get this iteration merged first. 
```suggestion
outputs:
  - name: out
    type: Directory
    glob: output*
```